Cargo.toml

```diff
@@ -1,22 +1,20 @@
 [workspace]
 resolver = "2"
 members = [
-    "fluent-syntax",
-    "fluent-bundle",
-    "fluent-fallback",
-    "fluent-resmgr",
-    "fluent-pseudo",
-    "fluent-testing",
-    "fluent",
-    "intl-memoizer"
+  "fluent-syntax",
```

Comment:
Please don't include all the unrelated re-styling in this PR, it is duplicated by #348 anyway.

Reply:
got it, this rebase should address your concerns